src/payment/payment-method.ts

```diff
@@ -12,4 +12,5 @@ export default interface PaymentMethod {
     nonce?: string;
     initializationData?: any;
     returnUrl?: string;
+    strategyVersion?: string;
```

Comment:
Also might be worth avoiding having a new `strategyVersion` param. You could just simple have a strategy called `klarnav2`. That should simplify the PR quite a bit.

Reply:
Actually I have created a new strategy called `klarnav2` but this is the first approach where does not exists a module for `klarnav2` so I can not choose the strategy with the `gatewayId` nor with `providerId` so I had to implement another way to choose the strategy that is why I add this new field. Also you can see this [comment](https://github.com/bigcommerce/checkout-sdk-js/pull/737#issuecomment-552703673). Further more right now I'm working on another approach that is about using same strategy and module for both versions of `Klarna`.

Reply:
wouldnt it be possible to call in the SDK the methodID `klarnav2`?

```js
// PaymentStrategyType.KLARNAV2 = 'klarnav2'
 registry.register(PaymentStrategyType.KLARNAV2, () =>
        new KlarnaPaymentV2Strategy(
            store,
            orderActionCreator,
            paymentMethodActionCreator,
            remoteCheckoutActionCreator,
            new KlarnaScriptLoader(scriptLoader)
        )
    );
```

This way you use a different class and avoid having if conditions in the main class. You could always make `KlarnaPaymentV2Strategy extends KlarnaPaymentStrategy` and internally just reuse the same logic.